### ISSUE TITLE
The database hub operator run error

### ISSUE THREAD

**Issue by wallyell** [plynx-team/plynx#122]
The database hub operator which defined by plynx.plugins.hubs.collection.CollectionHub run with error as below：

Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/plynx-1.7.4-py3.7.egg/plynx/plugins/executors/python/dag.py", line 99, in worker_main
    pickled_fn_parameter = node.get_parameter_by_name("_pickled_fn")
  File "/usr/local/lib/python3.7/site-packages/plynx-1.7.4-py3.7.egg/plynx/db/node.py", line 342, in get_parameter_by_name
    return self._get_custom_element(self.parameters, name, throw)
  File "/usr/local/lib/python3.7/site-packages/plynx-1.7.4-py3.7.egg/plynx/db/node.py", line 332, in _get_custom_element
    raise Exception('Parameter "{}" not found in {}'.format(name, self.title))
Exception: Parameter "_pickled_fn" not found in Numbers A to B

**Comment by wallyell** [comment]
The db.node init with out parameter _pickled_fn bug dag init with this parameter, it seems like a bug.

**Comment by khaxis** [comment]
Thanks @wallyell for reporting it!

Probably this is not a bug exactly... Or more precisely it's a bug in the config

The workflow you are using is defined by`executor: plynx.plugins.executors.python.dag.DAG` is not compatible with operations in the hub.

```
    - kind: db-hub
      title: Database hub
      icon: feathericons.database
      cls: plynx.plugins.hubs.collection.CollectionHub
      args:
        operations:
          - basic-file
          - basic-bash-jinja2-operation
          - basic-python-node-operation
          - basic-dag-operation
        collection: templates
``` 
None of those ^ operations work with `python-workflow` in the config.

**Comment by khaxis** [comment]
Yes, it should be fixed with in next release. Meanwhile, is it a blocking issue? I would be happy to assist with it!

**Comment by wallyell** [comment]
Thanks, it solves my problem by updating config file.

**Comment by frank-io** [comment]
@wallyell Hi, I think i had the same issue while trying to run a python workflow with a `Python Code Operation` which was created from the UI. Could you pls advice how did you fix the issue? Appreciated

**Comment by khaxis** [comment]
@frank-io 
should be fixed in 1.7.9 https://github.com/plynx-team/plynx/pull/128/files#diff-084b3435c47f3672bc5888b44fc556fc598132bca1d13657501646aad1a6f1f9R142 You will need to update the config (or `cp template_config.yaml config.yaml`)

The reason: the Workflow used here did not support the Operations defined in the UI. It was the matter of transforming the backend a little done recently
